packages/mobile/src/index.d.ts

```diff
@@ -6,7 +6,6 @@ declare module '*.json' {
 declare module '*.png'
 declare module '*.jpg'
 
-declare module 'react-native-geth'
```

Comment:
?

Reply:
We were declaring an empty module here because we didn't have typing info for the `react-native-geth` so this was just a way to make typescript shut up. But this branch is based on my branch in `react-native-geth` that now exports types which we get on import and no longer need this.